src/content/development/backports/_index.en.md

```diff
@@ -30,6 +30,7 @@ Backporting a pull request (PR) is automated by running:
 
 `make LOCAL_BUILD=1 backport release=<release-branch> pr=<PR to cherry-pick>`
 
+Please ensure that [shipyard](https://github.com/submariner-io/shipyard) repo is checked out alongside the project.
```

Comment:
Can you help me understand what problem this is trying to help people avoid? I tried a backport in the main repo without doing any special checkout of Shipyard and it worked fine. I do have both the main repo and Shipyard in my `~/go/src/submariner-io` dir, is that what you mean is required?

Reply:
Yes that's correct @dfarrell07. The shipyard repo should be alongside (i.e., $PWD/..) the current project.